generators/php/base/src/asIs/Client/RawClient.Template.php

```diff
@@ -178,17 +179,28 @@ private function encodeRequestBody(
     private function buildJsonBody(
         mixed $body,
         array $options,
-    ): mixed {
+    ): mixed
+    {
         $overrideProperties = $options['bodyProperties'] ?? [];
+        if(is_array($body) && self::isSequential($body)){
```

Comment:
nit: Let's tidy up the spacing on these:
```suggestion
        if (is_array($body) && self::isSequential($body)) {
```